spec/queries/content_items_query_spec.rb

```diff
@@ -43,14 +43,39 @@
       end
     end
 
-    describe "filtering by organisation" do
-      let(:organisation) { create(:organisation, content_items: content_items) }
+    describe "filtering" do
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/6ce7b40c87eac95cc7d5022454a7755d757dee11